config/cypress.webpack.config.js

```diff
@@ -20,32 +20,40 @@ const { config: webpackConfig, plugins } = config({
   }),
 });
 
-plugins.push(
-  new webpack.DefinePlugin({
-    IS_DEV: true,
-  })
-);
-
-// required to mock the chrome functionss
-webpackConfig.module.rules.push({
+module.exports = {
+  ...webpackConfig,
+  plugins: [
+    ...plugins,
+    new webpack.DefinePlugin({
+      IS_DEV: process.env.NODE_ENV !== 'production',
+    }),
+  ],
+  module: {
+    ...webpackConfig.module,
+    rules: [
+      ...webpackConfig.module.rules,
+      {
+        resolve: {
+          alias: {
+            '@redhat-cloud-services/frontend-components/useChrome': resolve(
+              __dirname,
+              './overrideChrome.js'
+            ),
+            '../useChrome': resolve(__dirname, './overrideChrome.js'),
+          },
+        },
+      },
+      {
+        test: /\.(?:js|mjs|cjs)$/,
+        exclude: /(node_modules|bower_components)/i,
+        use: ['babel-loader'],
+      },
+    ],
+  },
   resolve: {
-    alias: {
-      '@redhat-cloud-services/frontend-components/useChrome': resolve(
-        __dirname,
-        './overrideChrome.js'
-      ),
-      '../useChrome': resolve(__dirname, './overrideChrome.js'),
+    fallback: {
+      stream: require.resolve('stream-browserify'),
+      zlib: require.resolve('browserify-zlib'),
```

Comment:
Could you provide a brief background on why this is added?

Reply:
I just took what was in compliance configured. I'll try and see if and why this would be required.